Podverse/PVMediaRefRetriever.swift

```diff
@@ -0,0 +1,70 @@
+//
```

Comment:
Maybe make this function a static function on MediaRef class instead. No need for a whole extra class for one method directly related to mediaRefs